Development environment setup/Xdebug.md

```diff
@@ -0,0 +1,101 @@
+Xdebug is an indispensible tool when working on PHP projects. It's a very powerful debugger which can be used for code coverage when running tests as well.
+
+If you have Laravel Herd Pro installed, Xdebug is already installed and configured for you. If you don't have Laravel Herd Pro, you can install Xdebug manually.
```

Comment:
This should be changed to mention that Herd includes Xdebug in the free version as well, but it just requires a tiny bit of config.